### ISSUE TITLE
Can't uncheck displayed columns

### ISSUE THREAD

**Issue by aerni** [statamic/cms#7833]
### Bug description

It's impossible to uncheck displayed columns in the listing view as the whole item is draggable.

![CleanShot 2023-04-05 at 10 40 35](https://user-images.githubusercontent.com/23167701/230028797-c62134b9-2d00-4110-accd-e57b4bb86e58.gif)

### How to reproduce

Install [v4.0.0-alpha.2](https://github.com/statamic/cms/releases/tag/v4.0.0-alpha.2) and follow the video.

### Environment

```yaml
Environment
Application Name: Statamic
Laravel Version: 9.52.5
PHP Version: 8.1.13
Composer Version: 2.5.0
Environment: local
Debug Mode: ENABLED
URL: statamic-v4.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 0
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.0.0-alpha.2 Solo
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

None

**Comment by jonassiewertsen** [comment]
I can confirm, as I just wanted to report the same. Aerni has been faster though :-)

**Comment by jasonvarga** [comment]
Fixed in #7829
